frontend/src/app/components/ErrorPage.js

```diff
@@ -7,7 +7,7 @@ export default class ErrorPage extends React.Component {
   render() {
     return (
       <div className="full-page-wrapper space-between">
-        <Header />
+        <Header openWindow={this.props.openWindow} sendToGA={this.props.sendToGA} />
```

Comment:
Hmm, I was a little reticent to do that, since it only needs these 2 props. But, that could be easier as boilerplate